snapshotter/cmd/devmapper/README.md

```diff
@@ -0,0 +1,64 @@
+# devmapper snapshotter for containerd and Firecracker
+
+This component is a
+[snapshotter](https://github.com/containerd/containerd/blob/master/design/snapshots.md)
+[plugin](https://github.com/containerd/containerd/blob/master/PLUGINS.md) for
+containerd that stores snapshots in ext4-formatted filesystem images in a
+devicemapper thin pool.  The snapshots created by this snapshotter are usable
+with the containerd-firecracker-runtime to run microVM-backed containers with
+the Firecracker VMM.
+
+## Installation
+
+To make containerd aware of this plugin, you need to register it in
+containerd's configuration file.  This file is typically located at
+`/etc/containerd/config.toml`.
+
+Here's a sample entry that can be made in the configuration file:
+
+```toml
+[proxy_plugins]
+  [proxy_plugins.firecracker-dm-snapshotter]
+    type = "snapshot"
+    address = "/var/run/firecracker-dm-snapshotter.sock"
+```
+
+The name of the plugin in this example is "firecracker-dm-snapshotter".  The
+`address` entry points to a socket file exposed by the snapshotter, which is
+determined when you run it.
+
+## Usage
+
+```
+./devmapper_snapshotter -address UNIX-DOMAIN-SOCKET -path ROOT -debug
```

Comment:
We don't use `-path` for devmapper (we have optional `-config` instead)

Reply:
Oops, missed that...